### PR TITLE
fix: dockerize dev environment

### PR DESCRIPTION
**Pull request by JellyBellyDev**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,69 +1,58 @@
-version: '3.8'
+version: "3.1"
```
Comment (garak) [inline docker-compose.yml:1]: Why the downgrade?

**Review by garak — CHANGES_REQUESTED** [review]

**Review by garak — APPROVED** [review]

**Review by AlessandroMinoccheri — APPROVED** [review]
LGTM 👍

File: Makefile [hunk 2]
```diff
@@ -18,17 +18,17 @@ run: ## run app
...
+	- docker-compose up --build -d
 
 stop: ## stop docker containers
 	- docker-compose down
 
-dc_build: ## rebuild docker compose containers
+dc_build_prod: ## rebuild docker compose containers
 	- make .docker_img_deps
-	- docker-compose up --build
+	- docker-compose -f docker-compose.prod.yml up --build
```
Comment (fabiocicerchia) [inline Makefile:31]: we don't have docker swarm on prod, having the prod version will not change the way we deploy to prod.

File: docker-compose.prod.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+version: '3.8'
```
Comment (fabiocicerchia) [inline docker-compose.prod.yml:1]: not needed, we use ECS and everything is managed by cloudformation.

File: docker-compose.yml [hunk 4]
```diff
@@ -1,69 +1,58 @@
-version: '3.8'
+version: "3.8"
 
 services:
   nginx:
-    build:
-      context: .
-      dockerfile: sys/docker/alpine-nginx/Dockerfile
-      network: host
+    image: nginx:alpine
```
Comment (fabiocicerchia) [inline docker-compose.yml:5]: this is going to diverge from the actual image we use on prod: `fabiocicerchia/nginx-lua`
it'd be better to use at least the same images, if not the same dockerfile.

File: sys/dev/nginx/nginx.conf [hunk 5]
```diff
@@ -0,0 +1,49 @@
+server {
```
Comment (fabiocicerchia) [inline sys/dev/nginx/nginx.conf:1]: not a big fan of having different configurations for dev and prod. for me, they should match 1:1.

File: sys/dev/php-fpm/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,52 @@
+FROM php:8.0-fpm-alpine
```
Comment (fabiocicerchia) [inline sys/dev/php-fpm/Dockerfile:1]: not a big fan of having different configurations for dev and prod. for me, they should match 1:1.

**Pull request merged**